README.md

```diff
@@ -14,6 +14,15 @@ $ tns plugin add nativescript-iqkeyboardmanager
 
 That's it! IQKeyboardManager takes care of all initialization when your app starts up by default.
 
+## Advanced usage
+
+Want to tweak the appearance and/or behavior?
+
+Take a look at the embedded [demo app](demo/) for a bunch of features.
+You can easily access [any other feature](https://github.com/hackiftekhar/IQKeyboardManager/blob/df3afc60ad92a2264dc5e101a3c7966d7f6265ef/PROPERTIES%20%26%20FUNCTIONS.md) because this plugin exposes the entire library through [a TypeScript definition file](demo/references.d.ts).
```

Comment:
This wording makes it seem like the demo’s `references.d.ts` file is the library’s TypeScript definition file.

Could you remove the link, and then add a sentence that says something like... _To use the definition file in your project, just add an entry to your `tsconfig.json` [like this]()._